### DISCUSSION TITLE
Vercel Analytics keep awaiting data

### DISCUSSION THREAD

**Discussion by iShawnWang** [vercel/next.js#24043] [Ideas]
### What version of Next.js are you using?

10.1.3

### What version of Node.js are you using?

Vercel

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

I enable Vercel Analytics, and redeploy, it keep awaiting data.
After inspect, the analytics request keep pending ? Even in Chrome incognito window
I don't know why, or should i provide more infomation ? This is really wired !

![image](https://user-images.githubusercontent.com/20266594/114661227-3e565f80-9d29-11eb-9f45-83f20376c924.png)

### Expected Behavior

I can view Analytics data

### To Reproduce

The project facing this problem:  https://github.com/iShawnWang/iShawnWang2333

**Comment by ashleyisles** [comment]
Was this resolved? I'm experiencing something similar.

**Reply by rswerve** [reply]
Not sure if it would make a difference, but this should probably be classified under "Help" rather than "Ideas."

I'm also seeing this issue.  Site deployed on Vercel, no analytics after successfully enabling them.

**Comment by ismaelrumzan** [comment]
Hi there, apologies for the challenge you are facing with analytics. Could you please review this [link](https://vercel.com/docs/concepts/analytics#enabling-the-feature) and follow the steps outlined there?

**Reply by rswerve** [reply]
For what it's worth, analytics started working for me today, without my having changed anything. 🤷

**Reply by rmi111** [reply]
I am facing issue with Vercel analytics same as others, followed the steps and still "Awaiting Data" status. Nothings working.

**Comment by brettgrigsby** [comment]
I also have this issue. The analytics tab for my app still says `Awaiting Data...`. When I visit the site, I can see the successful call to the analytics api endpoint:

```
Request URL: https://<REDACTED>/_vercel/insights/view
Request Method: POST
Status Code: 200
```

**Comment by golam71** [comment]
The Vercel Analytics page will initialize after a single data point is collected in the [time range](https://vercel.com/docs/concepts/analytics/limits-and-pricing) you're looking at, and at most after 30 minutes has elapsed. For more information on when data points are collected, please check out our [Analytics Documentation](https://vercel.com/docs/concepts/analytics/privacy#how-the-data-points-are-tracked).

[source](https://vercel.com/guides/how-many-data-points-do-i-need-to-initialize-vercel-analytics)

**Comment by miltonbolonha** [comment]
Same here, 3 months now.

**Comment by bill1on** [comment]
Had this issue too, for me it was because I had UBlock origin installed as an extension which blocked some of the traffic, after disabling it for my site everything went smoothly.

**Comment by edwardshturman** [comment]
Hi all, +1-ing this; uBlock Origin definitely yells at anything from `/_vercel/insights/`. I read up on related Issues and agree with the sentiment there that adblockers are too prominent (particularly uBlock) to ignore this. Would love to help come up with a solution!
